src/index.ts

```diff
@@ -8,12 +8,17 @@ import { Exporter } from "./exporter"
 
 const program = new Command("askar-tools-javascript")
 
+export enum Strategy {
+  EXPORT = 'export',
+  DB_PW_EXPORT = 'db-pw-export'
+}
+
```

Comment:
I think we have to export wallets based on the database scheme instead of creating our we can reuse the same